config/initializers/smtp.rb

```diff
@@ -0,0 +1,9 @@
+ActionMailer::Base.smtp_settings = {
+  address: 'smtp.sendgrid.net',
+  port: 587,
+  domain: 'localhost',
```

Comment:
in the future maybe we'll want to move this to env variables to be able to have working links in both local development and production server. Lets for now keep it as it is as we're not deploying the app anywhere just yet.